src/Context/CartContext.tsx

```diff
@@ -0,0 +1,84 @@
+import { createContext, useContext, useState, useEffect } from 'react';
+
+
+interface Product {
+  id: number;
+  title: string;
+  artist: string;
+  price: number;
+  image_url: string;
+  quantity: number;
+}
```

Comment:
```suggestion
import { Vinyl } from '../utils/types.ts';
```
This interface should be imported from the types @JoshCodedit added for all database schemas